src/main/java/TaskManager.java

```diff
@@ -0,0 +1,61 @@
+public class TaskManager {
+    public Task[] taskList = new Task[100];
+    public int numOfTasks = 0;
+
+    /**
+     * Add a task into the list
+     *
+     * @param input task to add
+     */
+    public void addTask(String input){
+        taskList[numOfTasks] = new Task(input);
+        numOfTasks++;
+        System.out.println(
+                Duke.PRINT_LINE
+                + "added: " + input + "\n"
+                + Duke.PRINT_LINE
+        );
+    }
+
+    /**
+     * mark a task in the list
+     *
+     * @param input position of item to mark in the list
+     */
+    public void markTask(int input){
+        if(input > 0 && input <= numOfTasks){
```

Comment:
This if-else violates the code standards in terms of spacing. It should be: `if (input > 0 && input <= numOfTasks) {`